tests/03-auth.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { chai } from 'mochainon';
+import { ResinAuth } from '../lib/auth';
+import { TokenType } from '../lib/token';
+import apiKeyFixtures from './fixtures/api-keys';
+import jwtFixtures from './fixtures/jwts';
+
+const { expect } = chai;
+const IS_BROWSER = typeof window !== 'undefined';
+
+let dataDirectory;
+if (!IS_BROWSER) {
+	// tslint:disable-next-line no-var-requires
+	const settings = require('resin-settings-client');
+	dataDirectory = settings.get('dataDirectory');
```

Comment:
TBH this part remains exactly as it is in [resin-token](https://github.com/resin-io-modules/resin-token/blob/master/tests/token.spec.ts#L11) so I think it's acceptable? What do you think?

Reply:
I'm sure there was somewhere where we changed it, to solve exactly this issue... Even if we haven't, I'm reasonably sure that's the effect of this, and it will be kind of annoying that every time you run these tests they log you out of the CLI, for example.

Reply:
I agree. Instead of changing the `dataDirectory` I changed the `tokenKey` so they are stored in the same directory with a different key